terraform/modules/fargate-task-definition/ecs.tf

```diff
@@ -61,5 +61,5 @@ resource "aws_ecs_task_definition" "td" {
 
 resource "aws_cloudwatch_log_group" "lamda_logs" {
```

Comment:
lamda? London Academy of Music and Dramatic Arts?

Reply:
That wasn't even my change!

Reply:
@IanUtley It's my misspelling, please direct any issues you might have to me and I will make sure to reply within 30 days